tests/unit/test_tf_estimator.py

```diff
@@ -305,6 +307,49 @@ def test_create_model_with_optional_params(sagemaker_session):
     assert model.vpc_config == vpc_config
 
 
+@patch("sagemaker.tensorflow.estimator.TensorFlow.create_model")
+def test_transformer_creation_with_endpoint_type(create_model, sagemaker_session):
+    tf = TensorFlow(
+        entry_point=SCRIPT_PATH,
+        role=ROLE,
+        sagemaker_session=sagemaker_session,
+        train_instance_count=INSTANCE_COUNT,
+        train_instance_type=INSTANCE_TYPE,
+    )
+    tf.latest_training_job = _TrainingJob(sagemaker_session, JOB_NAME)
+
+    tf.transformer(
+        INSTANCE_COUNT, INSTANCE_TYPE, model_server_workers=2, endpoint_type="tensorflow-serving"
+    )
+    create_model.assert_called_with(
+        endpoint_type="tensorflow-serving",
+        model_server_workers=2,
+        role="Dummy",
```

Comment:
use `ROLE` so that way if someone were to change what that variable was, they wouldn't have to search through the tests to find this hardcoded string